### PR TITLE
New site design

### PR DESCRIPTION
**Pull request by LuisRodriguezLD**
I think is about time we release this. What do you think @kusma?

### CODE REVIEW

**Comment by kusma** [comment]
@LuisRodriguezLD: would you mind rebasing this on master? There's some conflicts...

**Comment by AndrewEQ** [comment]
Great job man, I'm lovin it!

File: _includes/head.html [hunk 1]
```diff
@@ -6,7 +6,5 @@
   <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
   <link href="https://fonts.googleapis.com/css?family=Lato:700,900" rel="stylesheet">
   <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe" crossorigin="anonymous"></script>
-  {%- if jekyll.environment == 'production' and site.google_analytics -%}
-    {%- include google-analytics.html -%}
-  {%- endif -%}
+  {%- include analytics.html -%}
```
Comment (kusma) [inline _includes/head.html]: Why do you need this change? Can't we use the pre-existing integration?

File: _includes/icons.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<link rel="icon" href="{{ site.baseurl }}/assets/icons/favicon-32.png" sizes="32x32">
+
+<!-- For IE10 Metro -->
+<meta name="msapplication-TileColor" content="#FFFFFF">
+<meta name="msapplication-TileImage" content="{{ site.baseurl }}/assets/icons/favicon-144.png">
+<meta name="theme-color" content="#ffffff">
+
+<!-- Chrome for Android -->
+<link rel="manifest" href="{{ site.baseurl }}/assets/icons/manifest.json">
+<link rel="icon" sizes="192x192" href="{{ site.baseurl }}/assets/icons/favicon-192.png">
```
Comment (kusma) [inline _includes/icons.html]: This seems to be in the wrong commit... it's not related to analytics, no?

File: _includes/head.html (hunk 1 above)
Comment (kusma) [reply _includes/head.html]: I think that file is provided by the extension...

Comment (LuisRodriguezLD) [reply _includes/head.html]: To be honest, I am not sure. Should I revert the analytics then?

Comment (kusma) [reply _includes/head.html]: I don't think changing the analytics belong together with a redesign, no...

**Comment by kusma** [comment]
OK, I think this is so much better than the current site, that I think we should just merge it as-is. We can fix problems later!

**Pull request merged**